src/audio-worklet/free-queue/example/js/script.js

```diff
@@ -0,0 +1,71 @@
+// ***** Tests *****
```

Comment:
Remove the current comment, and add an overview comment explaining what's being tested and the expected result.

Reply:
Also follow the JS style guide:
https://github.com/google/eslint-config-google

Reply:
Sure!

Reply:
I don't see the change. Am I missing something?